packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2171,10 +2171,20 @@ export default class Meeting extends StatelessWebexPlugin {
       // LOCAL - check if we started sharing content
       else if (
         this.selfId === contentShare.beneficiaryId &&
-        contentShare.disposition === FLOOR_ACTION.GRANTED
+        contentShare.disposition === FLOOR_ACTION.GRANTED &&
+        contentShare.deviceUrlSharing === this.deviceUrl
       ) {
         // CONTENT - sharing content local
         newShareStatus = SHARE_STATUS.LOCAL_SHARE_ACTIVE;
+      } else if (
+        this.selfId === contentShare.beneficiaryId &&
+        contentShare.disposition === FLOOR_ACTION.GRANTED &&
+        contentShare.deviceUrlSharing !== this.deviceUrl
+      ) {
+        // CONTENT - same user sharing content from another client
+        // comparing deviceUrl to check if same user is sharing content from another client (eg native client)
+        // then send remote share activity to rest of the clients.
```

Comment:
"then send remote share activity to rest of the clients" - that comment is slightly incorrect, we don't send anything to the rest of the clients, we just send a notification to the SDK client app